.travis.yml

```
@@ -185,6 +185,14 @@ env:
       SERVER_PROCESS="DotNetRuntime.dll"
       IMAGE=openruntimes/dotnet:${VERSION}-6.0
       ARCH=linux/amd64,linux/arm64
+    
+    #Go
```

Let's make this match the others:

```suggestion
    # Go
```